packages/nimble-components/src/table/specs/table-column-width-hld.md

```diff
@@ -0,0 +1,157 @@
+# Table Column Widths
```

Comment:
Could you add a link to this from the main table spec? It'd also be worth a pass at the custom column spec to make sure it's not egregiously out of date and/or link to this too.